docs/references/qa/CometBFT-QA-v1.md

```diff
@@ -0,0 +1,307 @@
+|:--------------:|:--------------:|
+| ![latency-1-400-v38](img38/200nodes/e_de676ecf-038e-443f-a26a-27915f29e312.png) | ![latency-1-400-v1](imgs/v1/200nodes/latencies/e_8e4e1e81-c171-4879-b86f-bce96ee2e861.png) 
+| | ![latency-1-400-v1-le](imgs/v1/200nodes_with_latency_emulation/latencies/e_8190e83a-9135-444b-92fb-4efaeaaf2b52.png)
+
+In both cases, most latencies are around or below 4 seconds. On `v0.38` there are peaks reaching 10
+seconds, while on `v1` (without LE) the only peak reaches 8 seconds. In general, the images are
+similar. From this small experiment we infer that the latencies measured on `v1` are not worse than
+those of the baseline. With latency emulation, the latencies are considerably higher, as expected.
+
+### Metrics
+
+In this section we analyse key metrics extracted from Prometheus data on the 200-nodes experiment
+with configuration `c=1,r=400`.
+
+#### Mempool size
+
+The following figures show the evolution of the average and maximum mempool size over all full
+nodes. 
+
+**Average size** On `v1`, the average mempool size mostly stays below 1000 outstanding transactions
+except for a peak above 2000, coinciding with the moment the system reached round number 1 (see
+below). For these particular runs, this result is better than the baseline, which oscilates between
+1000 and 2500.
+
+With latency emulation, the average mempool size stays mostly above 2000 outstanding transactions
+with peaks almost reaching the maximum mempool size of 5000 transactions.
+
+| v0.38 | v1 (without LE / with LE) 
+| :--------------:|:--------------:|
+| ![mempool-avg-baseline](img38/200nodes/avg_mempool_size_ylim.png) | ![mempool-avg](imgs/v1/200nodes/metrics/avg_mempool_size.png)
+| | ![mempool-avg-le](imgs/v1/200nodes_with_latency_emulation/metrics/avg_mempool_size.png)
+
+**Maximum size** The maximum mempool size indicates when one or more nodes have reached their
+maximum capacity in terms of the number of transactions they can hold. In version `v0.38`, it's
+apparent that most of the time, at least one node that is dropping incoming transactions.
+Conversely, in `v1`, this happens less often, especially after reaching round 1 (as detailed below).
+
+However, when we introduce latency emulation into `v1`, there is consistently at least one node with
+a saturated mempool.
+
+| v0.38 | v1 (without LE / with LE)
+| :--------------:|:--------------:|
+| ![mempool-cumulative-baseline](img38/200nodes/mempool_size_max.png) | ![mempoool-cumulative](imgs/v1/200nodes/metrics/mempool_size_max.png)
+| | ![mempoool-cumulative-le](imgs/v1/200nodes_with_latency_emulation/metrics/mempool_size_max.png)
+
+#### Peers
+
+On all expertiments, the number of peers was stable on all nodes. As expected, the seed nodes have
+more peers (around 125) than the rest (between 20 and 70 for most nodes). The red dashed line
+denotes the average value. Just as in the baseline, the fact that non-seed nodes reach more than 50
+peers is due to [\#9548].
```

Comment:
@cason Rather than mentioning tendermint/tendermint#9548 here, can we cite #486 instead?

Reply:
Sure, we should.